portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,43 +14,62 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.*;
 import com.google.gson.Gson;
+import com.google.sps.data.Comment;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.annotation.WebServlet;
 import java.util.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private ArrayList<String> funMessages;
+    private ArrayList<Comment> commentHistory = new ArrayList<>();
 
-    public void init() {
-        funMessages = new ArrayList<>();
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query("Comment");
+        DatastoreService database = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = database.prepare(query);
+        
+        for (Entity entity : results.asIterable()) {
+            long id = entity.getKey().getId();
```

Comment:
I'm not sure what `id` corresponds to; do you know how it gets set when someone leaves a comment?  I'm guessing maybe it is just a numeric counter of #comments, but I'm not sure.

Reply:
Sorry for the delay. Yes it's just a way of enumerating them until I come up with a different way to do so.